azure-functions/acbs-function/mappings/facility/helpers/get-maximum-liability.test.js

```diff
@@ -0,0 +1,121 @@
+const getMaximumLiability = require('./get-maximum-liability');
+
+describe('getMaximumLiability', () => {
+  describe('when formatting the input', () => {
+    it('transforms a string into a number', () => {
+      const facility = {
+        amendment: {
+          amount: '789',
+        },
+      };
+
+      const result = getMaximumLiability(facility);
+      expect(result).toEqual(789);
+    });
+    it('transforms a string with commas into a number', () => {
+      const facility = {
+        amendment: {
+          amount: '78,910',
+        },
+      };
+
+      const result = getMaximumLiability(facility);
+      expect(result).toEqual(78910);
+    });
+  });
+
+  describe('when formatting the output', () => {
+    it('returns a number with 2 decimal places', () => {
```

Comment:
Should this be "returns the result of `to2Decimals`"? Instead of manual checking the result of the helper function here

Reply:
It _should_ technically, and it should technically mock the `to2Decimals` helper... In this case however as it's so small a helper, I think we're safe to just do this if thats okay?

Reply:
Just feels like it's a duplicated test, i.e it's testing what `to2Decimals` returns, but this function just returns the result of `to2Decimals`?

In other words, if `to2Decimals` changes, this test also needs to be updated, but it's purely consuming `to2Decimals`?

Reply:
agree, removed